packages/unmock-core/src/service/index.ts

```diff
@@ -3,6 +3,7 @@ import {
   ExtendedHTTPMethod,
   HTTPMethod,
   isRESTMethod,
+  StateStoreType,
```

Comment:
I still don't like the name `Store` for the exposed services, to me "store" is something else. But I don't really have a better alternative, maybe `StateFacade` or something but not sure that really matters at all.

Reply:
Sure, naming is often my weak spot.